docs/shipping/Code/python.md

```diff
@@ -488,23 +411,26 @@ Ensure that you include the `user-agent` header in the format: `"user-agent=logz
     
```

Comment:
Let's remove the example of the dice roll from here and keep only the instrumentation part, as seen in the documentation for the other SDKs.